Box_intersection_d/include/CGAL/Box_intersection_d/segment_tree.h

```diff
@@ -35,6 +35,13 @@ namespace CGAL {
 
 namespace Box_intersection_d {
 
+namespace internal {
```

Comment:
does not seem used

Reply:
Not used, except in parallel mode you mean ?